docs/kitematic/find_issue.md

```diff
@@ -31,7 +31,7 @@ To find and claim an issue you want to work on:
 
     The comments on the issues describe the problem and can provide information for a potential solution.
 
-    For this exercise we will select the issue #xxxx Missing container id.
+    For this exercise we will select the issue <a href="https://github.com/kitematic/kitematic/issues/1191" target="_blank">#1191 Missing container id</a>.
```

Comment:
Same thing here, the proper link is:
`https://github.com/docker/kitematic/issues/1191`